### PR TITLE
Add debug logging before bind or search_as

### PR DESCRIPTION
**Pull request by stevecj**
Log at debug level before binding or searching. This serves 2 purposes:
1. It includes details on success that are not otherwise logged except on failure.
2. When tailing the log, and the server is not responding, indicates what operation is in progress.

### CODE REVIEW

**Comment by stevecj** [comment]
Interesting. I usually use single-quoted unless there is a reason to do otherwise. I'm surprised that `black` wants the opposite. I'll fix that when I have a minute.

File: django_auth_ldap/config.py [hunk 1]
```diff
@@ -144,6 +144,11 @@ def execute(self, connection, filterargs=(), escape=True):
 
         try:
             filterstr = self.filterstr % filterargs
+            logger.error(
+                "Invoking search_s('{}', {}, '{}')".format(
+                    self.base_dn, self.scope, filterstr
+                )
```
[the remark this answers is not in the document]
Comment (stevecj) [reply django_auth_ldap/config.py:150]: I was wondering about that but trying to follow the same pattern as similar existing code.

[the remark this answers is not in the document]
Comment (francoisfreitag) [reply django_auth_ldap/config.py:150]: https://github.com/django-auth-ldap/django-auth-ldap/pull/292

**Comment by stevecj** [comment]
Oops. I didn't realize that I hadn't already added the `logger.debug(f'Binding as {bind_dn}')` suggestion. I just did that.

File: django_auth_ldap/backend.py [hunk 2]
```diff
@@ -830,6 +830,7 @@ def _bind_as(self, bind_dn, bind_password, sticky=False):
         the life of this object. If False, then the caller only wishes to test
         the credentials, after which the connection will be considered unbound.
         """
+        logger.debug(f"Binding as {bind_dn}")
```
Comment (francoisfreitag) [inline django_auth_ldap/backend.py:833]: Following #292:
```suggestion
        logger.debug("Binding as %s", bind_dn)
```

**Comment by stevecj** [comment]
Per my comment on #292, do we want to use `%s` or `{}` as placeholders in template strings for logging? django-auth-ldap does not support an old enough version of Python to not understand the `{}` form which is a bit more flexible than the old style.

**Comment by francoisfreitag** [comment]
> Per my comment on #292, do we want to use `%s` or `{}` as placeholders in template strings for logging? django-auth-ldap does not support an old enough version of Python to not understand the `{}` form which is a bit more flexible than the old style.

The comment on #292 is not viewable.
Using `{}` requires additional configuration for the logger (https://docs.python.org/3/howto/logging-cookbook.html#use-of-alternative-formatting-styles), but I can’t see the benefit for this project?
I’m thinking the easiest approach will probably be prefered, unless there are advantages to using the `{}` style?

**Review by francoisfreitag — APPROVED** [review]

**Pull request merged**